Schema Statements.sql

```diff
@@ -0,0 +1,67 @@
+CREATE DATABASE  IF NOT EXISTS `products`;
+USE `products`;
+
+DROP TABLE IF EXISTS `category`;
+CREATE TABLE `category` (
+  `id` int NOT NULL,
+  `name` varchar(45) NOT NULL,
+  PRIMARY KEY (`id`)
+);
+
+
+INSERT INTO `category` VALUES
+  (1,'foods'),
+  (2,'drinks'),
+  (3,'entertainment'),
+  (4,'pets'),
+  (5,'baby'),
+  (6,'houseware'),
+  (7,'health'),
+  (8,'other');
+
+DROP TABLE IF EXISTS `unit`;
+CREATE TABLE `unit` (
+  `id` int NOT NULL,
+  `name` varchar(45) NOT NULL,
+  PRIMARY KEY (`id`)
+);
+
+INSERT INTO `unit` VALUES 
+  (0,'g'),
+  (1,'ml'),
+  (2,'each');
+
+
+DROP TABLE IF EXISTS `origin`;
+CREATE TABLE `origin` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  PRIMARY KEY (`id`)
+);
+
+INSERT INTO `origin` VALUES (1,'Woolworths'),
+(2,'Aldi'),
+(3,'Coles');
+
+
+DROP TABLE IF EXISTS `items`;
+CREATE TABLE `items` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  `origin_id` int NOT NULL,
+  `price` decimal(19,2) NOT NULL,
+  `fk_category` int NOT NULL,
+  `quantity` int NOT NULL,
+  `fk_unit` int NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `fk_category_idx` (`fk_category`),
+  KEY `fk_qtyType_idx` (`fk_unit`),
```

Comment:
Shouldn't this be `unit_idx`?

Reply:
Keys are completely removed now, so it doesn't matter anymore.